messages/env.compute.collaborator.add.md

```diff
@@ -0,0 +1,13 @@
+# summary
+
+Add heroku user to all compute environments.
+
+# description
+
+You must provide the '--heroku-user' required flag.
+
+# examples
+
+- Add a heroku user to all compute environments:
+
+  <%= config.bin %> <%= command.id %> --heroku-user heroku-user
```

Comment:
the example value should probably be an email address instead of `heroku-user`

Reply:
Is there a reason the user email is passed as a flag instead of just an argument like how it works with adding collaborator in the heroku CLI: https://devcenter.heroku.com/articles/collaborating#add-collaborators

Reply:
@friism That was the copy noted in the PRD. However, we can change it to what you referenced if you prefer that.

Reply:
@zwhitfield3 Let's keep the flag the way it is, since that's what's in the PRD, but please update the example to have an email for the value, instead of saying `heroku-user` twice.